src/java/org/apache/cassandra/index/sai/disk/PostingList.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.cassandra.index.sai.disk;
```

Comment:
nit: I guess there's nothing inherently disk-bound about this interface. There are also things like `MergePostingList` and `OrdinalPostingList`...makes me wonder if a `sai.postings` package would be useful to group them.

Reply:
I like this idea, it's something that has been worrying me a bit. Postings are a very important aspect of the index and having them separate and in one place is much tidier.